mulighetsrommet-api/src/main/resources/db/migration/V59__virksomhet_tabell.sql

```diff
@@ -0,0 +1,11 @@
+create table virksomhet
+(
+    organisasjonsnummer     text not null,
+    overordnet_enhet        text references virksomhet(organisasjonsnummer) on delete cascade,
+    navn                    text not null,
+    created_at              timestamp default now() not null,
+    updated_at              timestamp default now() not null,
```

Comment:
Vi har en del av disse `updated_at` i databasetabellene, men har vi noe logikk som faktisk sørger for å holde denne kolonnen oppdatert? I `R__triggers.sql` er dette definert triggers for dette for et par tabeller, men jeg mistenker at denne ikke matcher helt med antall tabeller som har en `update_at`-kolonne :thinking:

Reply:
Jeg har trodd det skjedde automatisk, men der har jeg kanskje gått på en smell haha

Reply:
Jeg fjerner de 👍